wasm/__init__.py

```diff
@@ -295,47 +305,53 @@ def spec_validate_set_global(C, x):
 def spec_validate_t_load(C, t, memarg):
     if len(C["mems"]) < 1:
         raise Exception("invalid")
+    # TODO: Next line *may* have a bug since int(t[1:2]) would always return
+    # either `3` or `6`, both of which would result in `tval // 8` always
+    # evaluating to `0`.
     tval = int(t[1:2])  # invariant: t has form: letter digit digit  eg i32
     if 2 ** memarg["align"] > tval // 8:
         raise Exception("invalid")
```

Comment:
Although, then it's surprising that this isn't constantly raising exceptions. Is this method actually getting called? Maybe it's not being tested.